e2e-tests/constants/routes/insurance/account.js

```diff
@@ -36,8 +36,13 @@ const SIGNED_OUT = `${INSURANCE_ROOT}/signed-out`;
 
 const SUSPENDED = {
   ROOT: SUSPENDED_ROOT,
+  EMAIL_SENT: `${SUSPENDED_ROOT}/email-sent`,
+  VERIFY_EMAIL: `${SUSPENDED_ROOT}/verify-email`,
+  VERIFY_EMAIL_LINK_EXPIRED: `${SUSPENDED_ROOT}/link-expired`,
 };
 
+const REACTIVATED = `${INSURANCE_ROOT}/account-reactivated`;
```

Comment:
can we keep this on the top with all the string consts? not a blocker

Reply:
Good shout, working on this route for the next PR, have already updated 👍